### PR TITLE
feat(tables): Render boolean values as switches

### LINKED ISSUE #8321: [tables] Boolean cells should use toggle buttons

### PR DESCRIPTION
**Pull request by ZaymonFC**
- Resolves #8321

### CODE REVIEW

File: packages/ui/react-ui-table/src/util/table-controls.ts [hunk 1]
```diff
@@ -2,26 +2,89 @@
...
+export const CONTROL_IDENTIFIERS = {
   checkbox: 'data-table-checkbox',
-  header: 'data-table-header',
+  switch: 'data-table-switch',
 } as const;
 
-export type RenderCheckboxProps = { rowIndex: number; header?: boolean; checked?: boolean; disabled?: boolean };
+const BASE_CLASSES = {
+  checkbox: 'absolute inset-block-[6px] inline-end-[6px] ch-checkbox',
+  switch: 'absolute inset-block-[4px] inline-end-[4px] ch-checkbox--switch',
```
Comment (thure) [inline packages/ui/react-ui-table/src/util/table-controls.ts:27]: Gorgeous ✨

Recommend using rem for these tho:
```suggestion
  checkbox: 'absolute inset-block-[.375rem] inline-end-[.375rem] ch-checkbox',
  switch: 'absolute inset-block-[.25rem] inline-end-[.25rem] ch-checkbox--switch',
```

File: packages/ui/react-ui/src/components/Input/Input.tsx [hunk 2]
```diff
@@ -279,63 +281,46 @@ const Checkbox: ForwardRefExoticComponent<CheckboxProps> = forwardRef<
...
-            {...(checked && {
-              weight,
-              className: tx('input.checkboxIndicator', 'input--checkbox__indicator', { size }),
-            })}
-          />
-        </CheckboxIndicatorPrimitive>
+        <Icon
+          icon={checked === 'indeterminate' ? 'ph--minus--regular' : 'ph--check--regular'}
+          classNames={tx('input.checkboxIndicator', 'input--checkbox__indicator', { size, checked })}
+        />
```
Comment (thure) [inline packages/ui/react-ui/src/components/Input/Input.tsx:277]: Thank you! ✨

Comment (ZaymonFC) [reply packages/ui/react-ui/src/components/Input/Input.tsx:277]: 🫡

**Review by thure — APPROVED** [review]
Brilliant ✨

**Pull request merged**